### PR TITLE
Add a helpful topic for new users

### PR DESCRIPTION
**Pull request by SueChaplain**
Users who are new to OpenJDK with an OpenJ9 VM
often ask what the differences are compared to
HotSpot. This topic helps to summarise the
differences and provide links to other useful
resources.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
@pshipton Please can you check this topic? I think you've probably seen all but the last section already. I've opted for listing some of the Oracle vs OpenJDK differences until there is something more concrete at the Adopt project (WIP). I'd like to get this topic in before we create the 0.12.0 tag.

File: docs/openj9_newuser.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+- [**Garbage Collection Memory Vizualizer (GCMV):**](https://marketplace.eclipse.org/content/ibm-monitoring-and-diagnostic-tools-garbage-collection-and-memory-visualizer-gcmv) Plots GC and native memory data over time. You can view and save data as a report, raw log, tabulated data, or in graphical format. The tool helps to diagnose problems such as memory leaks with data presented in various visual formats for analysis. Tuning recommendations are also provided. For more information, read the [GCMV documentation](https://www.ibm.com/support/knowledgecenter/en/SS3KLZ/com.ibm.java.diagnostics.visualizer.doc/homepage/plugin-homepage-gcmv.html).
+- [**Memory Analyzer:**](https://marketplace.eclipse.org/content/memory-analyzer-0) Examines the Java object heap to help find memory leaks or reduce memory consumption. Support is available for OpenJ9 via the DTFJ interface. More information about Eclipse MAT can be found on the [project website page](https://www.eclipse.org/mat/).
```
Comment (pshipton) [inline docs/openj9_newuser.md]: We should give instructions on how to load the DTFJ plugin into MAT. Afaik this information is only available in one of the release notes. https://www.eclipse.org/openj9/docs/version0.9/ `installing the DTFJ plugin. (Install from the Eclipse Help menu; Install New Software > Work with "IBM Diagnostic Tool Framework for Java" > IBM Monitoring and Diagnostic Tools > Diagnostic Tool Framework for Java)`

File: docs/openj9_newuser.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+**Java plug-in and webstart:**
+
+These features were deprecated by Oracle in Java SE 9 and are not available in Oracke JDK 11. Oracle encourages developers to migrate to other solutions.
+
+**T2K font library:**
+
+T2K is a proprietary native font library, which is licensed for use by Java 2D from a third party. OpenJDK uses the open FreeType font library as an alternative. The rendering of these fonts is almost identical and situations where there are differences are rare.
+
+Discussions are underway at the AdoptOpenJDK project to include OpenJFX and investigate the IcedTea-web project as a replacement for the Web Start capabilities.
```
Comment (pshipton) [inline docs/openj9_newuser.md]: This paragraph looks like it is part of the `T2K font library` topic, but it is related to JavaFX.

File: docs/openj9_newuser.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+In addition there are currently no installers or updaters available for OpenJDK binaries from AdoptOpenJDK.
```
Comment (pshipton) [inline docs/openj9_newuser.md]: This paragraph looks like it is part of the `T2K font library` topic. Perhaps it needs a title.

File: docs/openj9_newuser.md (hunk 1 above)
Comment (SueChaplain) [reply docs/openj9_newuser.md]: Done

File: docs/openj9_newuser.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+for memory leaks and to reduce memory consumption. For more information about DTFJ, see [Using the Diagnostic Tool Framework for Java](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/dtfj.html).
+- **java.lang.management API:** OpenJ9 provides MXBean additions and extensions to this standard API, which enables you to use tools such as JConsole to monitor and manage your Java applications. For more information, see [MBeans and MXBeans](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mxbeans.html).
+
+If you are familiar with using HotSpot as part of an Oracle JDK or OpenJDK, you might make use of certain `com.sun.management` interfaces. Although OpenJ9 implements some of these interfaces, a few are specific to the HotSpot VM. The following table indicates alternative classes or mechanisms that you can use for equivalent function in OpenJ9:
+
+| Hotspot-specific classes   | Alternatives for OpenJ9  |
+|----------------------------|--------------------------|
+|`HotSpotDiagnosticMXBean`   | `OpenJ9DiagnosticsMXBean`|
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/openj9_newuser.md]: Modified

File: docs/openj9_newuser.md (hunk 3 above)
Comment (SueChaplain) [reply docs/openj9_newuser.md]: Reorganized

**Pull request merged**